### PR TITLE
DM-44346: Add back in shape flag for trailed sources

### PR DESCRIPTION
**Pull request by bsmartradio**
Bug fix to add back in the shape flag for trailed sources.

### CODE REVIEW

File: python/lsst/meas/extensions/trailedSources/NaivePlugin.py [hunk 1]
```diff
@@ -134,6 +134,7 @@ def __init__(self, config, name, schema, metadata, logName=None):
         self.NAN = flagDefs.add("flag_nan", "One or more trail coordinates are missing")
         self.SUSPECT_LONG_TRAIL = flagDefs.add("flag_suspect_long_trail",
                                                "Trail length is greater than three times the psf radius")
+        self.SHAPE = flagDefs.add("flag_shape", "Shape flag is set, trail not calculated")
```
Comment (mrawls) [inline python/lsst/meas/extensions/trailedSources/NaivePlugin.py]: Do you mean trail **length** is not calculated?

File: python/lsst/meas/extensions/trailedSources/NaivePlugin.py [hunk 2]
```diff
@@ -166,8 +167,8 @@ def measure(self, measRecord, exposure):
         ra, dec = self.computeRaDec(exposure, xc, yc)
 
         if measRecord.getShapeFlag():
-            self.log.warning("Shape flag is set for measRecord: %s. Trail measurement "
-                             "will not be made.", measRecord.getId())
+            self.log.debug("Shape flag is set for measRecord: %s. Trail measurement "
+                           "will not be made.", measRecord.getId())
```
Comment (mrawls) [inline python/lsst/meas/extensions/trailedSources/NaivePlugin.py]: Can this log message be a bit more specific - which values will be set to nan, e.g.? Just length? Others also?

**Review by mrawls — APPROVED** [review]
Happy to have this downgraded to debug per Slack discussion, but please ensure Jenkins and ci_things are happy, because we all know how messing with flag columns can lead to exciting side effects.

File: python/lsst/meas/extensions/trailedSources/NaivePlugin.py (hunk 1 above)
Comment (bsmartradio) [reply python/lsst/meas/extensions/trailedSources/NaivePlugin.py]: Yes I do. Changed.

File: python/lsst/meas/extensions/trailedSources/NaivePlugin.py (hunk 2 above)
Comment (bsmartradio) [reply python/lsst/meas/extensions/trailedSources/NaivePlugin.py]: Added explanation that all trail values will be nan if the flag is set.

**Comment by bsmartradio** [comment]
> Happy to have this downgraded to debug per Slack discussion, but please ensure Jenkins and ci_things are happy, because we all know how messing with flag columns can lead to exciting side effects.

https://rubin-ci.slac.stanford.edu/blue/organizations/jenkins/stack-os-matrix/detail/stack-os-matrix/1516/pipeline. Everything's good.

**Pull request merged**